models/transformation/dme_claims.sql

```diff
@@ -1,137 +1,137 @@
 select
-    cast(h.cur_clm_uniq_id as varchar) as claim_id

```

Comment:
Replaced all varchar and date conversions to custom macros to handle the syntax/function differences between data warehouses.